### PR TITLE
Adding info about Scenario

### PR DESCRIPTION
**Pull request by ThomasLandauer**
See https://github.com/Codeception/Codeception/issues/6035

TODO: Update the anchor after https://github.com/Codeception/codeception.github.com/pull/378 is online

### CODE REVIEW

**Comment by ThomasLandauer** [comment]
The anchor is OK, so the above TODO is done.

File: src/Codeception/Module/WebDriver.php [hunk 1]
```diff
@@ -293,6 +293,20 @@
  * until it finally tries to find the form by XPath).
  * If speed is a concern, it's recommended you stick with explicitly specifying the locator type via the array syntax.
  *
+ * ### 
+ *
+ * You can switch between browsers inside your code by injecting `\Codeception\Scenario` into your test like this:
```
Comment (Naktibalda) [inline src/Codeception/Module/WebDriver.php:298]: This sentence doesn't make sense to me.

In order to change browsers you have to use
```php
$this->getModule('WebDriver')->_restart(['browser' => $browser]);
```
as documented at https://codeception.com/docs/modules/WebDriver#_restart

**Review by Naktibalda — CHANGES_REQUESTED** [review]

File: src/Codeception/Module/WebDriver.php (hunk 1 above)
Comment (ThomasLandauer) [reply src/Codeception/Module/WebDriver.php:298]: The idea was to show some example for scenario, see https://github.com/Codeception/Codeception/issues/6035#issuecomment-730257318 - and I couldn't think of another use case as switching the browser ;-)

I now added your line - but will `$this->getModule(...)` work from within a test at all? Or do I have to think of another use case...?

Comment (Naktibalda) [reply src/Codeception/Module/WebDriver.php:298]: `getModule` method is not accessible in test code, it can only be used in helpers.

**Comment by ThomasLandauer** [comment]
I deleted my fork, so I had to start from scratch :-) See https://github.com/Codeception/module-webdriver/pull/65

**Pull request closed without merging**